### PR TITLE
Fixed Typos on tour.md.

### PR DESCRIPTION
**Pull request by Jaxelr**
# Fixed typos on Tour of F#  doc.

## Summary

There was some minor typos on the Tour of F# doc.

## Details

Fixed typos on tour.md

### CODE REVIEW

File: docs/fsharp/tour.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ One special case of Discriminated Union types is the Option Type, which is so us
 
 One unique feature of F#'s type system is the ability to provide context for numeric literals through Units of Measure.
 
-[Units of Measure](language-reference/units-of-measure.md) allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a certain context, and eliminate runtime errors associated with that kind of work.
+[Units of Measure](language-reference/units-of-measure.md) allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a thus eliminating runtime errors associated with that kind of work.
```
Comment (svick) [inline docs/fsharp/tour.md]: This doesn't seem to be grammatically correct. Did you delete the "certain context" part by accident?

Comment (Jaxelr) [inline docs/fsharp/tour.md]: Woops, yes this was my mistake.

**Review by cartermp — CHANGES_REQUESTED** [review]
Thanks, @Jaxelr!  Looks good modulo the change @svick noted.  Once that's fixed, I'll go ahead and merge.

**Comment by mairaw** [comment]
Thanks @Jaxelr! I made the fixes for you so we can merge this. Thank you for catching and fixing those typos.

**Comment by mairaw** [comment]
@cartermp ok to merge this now?

**Review by cartermp — APPROVED** [review]

**Pull request merged**